util/pub_get_all

```diff
@@ -8,165 +8,63 @@ fi
-
-echo "\n### serverpod_test_server"
-cd $BASE/tests/serverpod_test_server
-dart pub upgrade
-
-echo "\n### serverpod_test_shared"
-cd $BASE/tests/serverpod_test_shared
-dart pub upgrade
-
-echo "\n### serverpod_test_module_client"
-cd $BASE/tests/serverpod_test_module/serverpod_test_module_client
-dart pub upgrade
-
-echo "\n### serverpod_test_module_server"
-cd $BASE/tests/serverpod_test_module/serverpod_test_module_server
-dart pub upgrade
-
-echo "\n### bootstrap_project"
-cd $BASE/tests/bootstrap_project
-dart pub upgrade
-
-echo "\n### serverpod_cli_e2e_test"
-cd $BASE/tests/serverpod_cli_e2e_test
-dart pub upgrade
-
-# Templates
-
-echo "\n### projectname_server"
-cd $BASE/templates/serverpod_templates/projectname_server
-dart pub upgrade
-
-echo "\n### projectname_client"
-cd $BASE/templates/serverpod_templates/projectname_client
-dart pub upgrade
-
-echo "\n### projectname_flutter"
-cd $BASE/templates/serverpod_templates/projectname_flutter
-flutter pub upgrade
-
-echo "\n### modulename_server"
-cd $BASE/templates/serverpod_templates/modulename_server
-dart pub upgrade
-
-echo "\n### modulename_client"
-cd $BASE/templates/serverpod_templates/modulename_client
-dart pub upgrade
-
-# Auth module
-
-echo "\n### serverpod_auth_server"
-cd $BASE/modules/serverpod_auth/serverpod_auth_server
-dart pub upgrade
-
-echo "\n### serverpod_auth_client"
-cd $BASE/modules/serverpod_auth/serverpod_auth_client
-dart pub upgrade
-
-echo "\n### serverpod_auth_shared_flutter"
-cd $BASE/modules/serverpod_auth/serverpod_auth_shared_flutter
-flutter pub upgrade
-
-echo "\n### serverpod_auth_apple_flutter"
-cd $BASE/modules/serverpod_auth/serverpod_auth_apple_flutter
-flutter pub upgrade
-
-echo "\n### serverpod_auth_google_flutter"
-cd $BASE/modules/serverpod_auth/serverpod_auth_google_flutter
-flutter pub upgrade
-
-echo "\n### serverpod_auth_email_flutter"
-cd $BASE/modules/serverpod_auth/serverpod_auth_email_flutter
-flutter pub upgrade
-
-echo "\n### serverpod_auth_firebase_flutter"
-cd $BASE/modules/serverpod_auth/serverpod_auth_firebase_flutter
-flutter pub upgrade
-
-# Chat module
-
-echo "\n### serverpod_chat_server"
-cd $BASE/modules/serverpod_chat/serverpod_chat_server
-dart pub upgrade
-
-echo "\n### serverpod_chat_client"
-cd $BASE/modules/serverpod_chat/serverpod_chat_client
-dart pub upgrade
-
-echo "\n### serverpod_chat_flutter"
-cd $BASE/modules/serverpod_chat/serverpod_chat_flutter
-flutter pub upgrade
-
-# Integrations
-
-echo "\n### serverpod_cloud_storage_s3"
-cd $BASE/integrations/serverpod_cloud_storage_s3
-dart pub upgrade
+# Check for --offline parameter
+if [ "$1" == "--offline" ]; then
+    OFFLINE_MODE="--offline"
+    echo "Running in offline mode"
+fi
 
-echo "\n### serverpod_cloud_storage_gcp"
-cd $BASE/integrations/serverpod_cloud_storage_gcp
-dart pub upgrade
+echo "PUB UPGRADE ALL"
 
-# Tooling
-echo "\n### serverpod_cli"
-cd $BASE/tools/serverpod_cli
-dart pub upgrade
\ No newline at end of file
+declare -a paths=(
```

Comment:
Suggestion: Split the list into two lists for `dart` and `flutter` packages to avoid the pattern matching in the path :) Should make it a bit more robust in the long run.